default-templates/server/index.html

```diff
@@ -30,6 +26,41 @@ <h1>Welcome to the Solid Prototype</h1>
 
     If you have not already done so, please create an account.
   </p>
+
+  <p class="lead hidden" id="loggedIn">
+    You are logged in as
+    <a href="#" id="profileLink"></a>.
+  </p>
 </div>
+<script src="/common/js/solid-auth-client.bundle.js"></script>
+<script>
+  const elements = {};
+  ['loggedIn', 'profileLink'].forEach(id => {
+    elements[id] = document.getElementById(id)
+  })
+
+  async function login () {
```

Comment:
Same as earlier: https://caniuse.com/#search=async

Reply:
Already in `default-views/auth/login-required.hbs`

Reply:
I'm sorry that I haven't raised concerns on this earlier, but I wasn't part of the review-process for the code that you refer to.

Reply:
Your concerns are definitely valid. It's just that other browsers are already broken by other pages, so this PR will not make a difference. Even if I'd make this page compatible with (let's say) IE9, the very next page they land on will not be.